### PR TITLE
[zh-cn] Update 'block formatting context'

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

start revamping block- and inline- glossary

### Related issues and pull requests

Part of #12632

### CODE REVIEW

File: files/zh-cn/web/guide/css/block_formatting_context/index.md [hunk 1]
```diff
@@ -151,27 +151,54 @@ section {
...
+
+.blue {
+  background: blue;
+}
+
+.red {
+  background: red;
+}
+```
+{{EmbedLiveSample("外边距重叠示例", 120, 170)}}
```
Comment (jasonren0403) [inline files/zh-cn/web/guide/css/block_formatting_context/index.md:186]:
```suggestion

{{EmbedLiveSample("外边距重叠示例", 120, 170)}}
```

File: files/zh-cn/web/guide/css/block_formatting_context/index.md [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
-- 表格标题（{{ cssxref("display") }} 值为 `table-caption`，HTML 表格标题默认值）
-- 匿名表格单元格元素（{{ cssxref("display") }} 值为 `table`、`table-row`、 `table-row-group`、`table-header-group`、`table-footer-group`（分别是 HTML table、tr、tbody、thead、tfoot 的默认值）或 `inline-table`）
-- {{ cssxref("overflow") }} 值不为 `visible`、`clip` 的块元素
-- {{ cssxref("display") }} 值为 `flow-root` 的元素
-- {{ cssxref("contain") }} 值为 `layout`、`content` 或 `paint` 的元素
-- 弹性元素（{{ cssxref("display") }} 值为 `flex` 或 `inline-flex` 元素的直接子元素），如果它们本身既不是 [flex](/zh-CN/docs/Glossary/Flex_Container)、[grid](/zh-CN/docs/Glossary/Grid_Container) 也不是 [table](/zh-CN/docs/Web/CSS/CSS_table) 容器
-- 网格元素（{{ cssxref("display") }} 值为 `grid` 或 `inline-grid` 元素的直接子元素），如果它们本身既不是 [flex](/zh-CN/docs/Glossary/Flex_Container)、[grid](/zh-CN/docs/Glossary/Grid_Container) 也不是 [table](/zh-CN/docs/Web/CSS/CSS_table) 容器
-- 多列容器（{{ cssxref("column-count") }} 或 {{ cssxref("column-width") }} 值不为 `auto`，包括`column-count` 为 `1`）
-- `column-span` 值为 `all` 的元素始终会创建一个新的 BFC，即使该元素没有包裹在一个多列容器中 ([规范变更](https://github.com/w3c/csswg-drafts/commit/a8634b96900279916bd6c505fda88dda71d8ec51), [Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=709362))
+- 文档级别的根元素（`<html>`）。
```
Comment (yin1999) [inline files/zh-cn/web/guide/css/block_formatting_context/index.md:12]:
```suggestion
- 文档的根元素（`<html>`）。
```

File: files/zh-cn/web/guide/css/block_formatting_context/index.md [hunk 3]
```diff
@@ -1,50 +1,50 @@
...
 格式化上下文影响布局，通常，我们会为定位和清除浮动创建新的 BFC，而不是更改布局，因为它将：
 
-- 包含内部浮动
-- 排除外部浮动
-- 阻止 [外边距重叠](/zh-CN/docs/Web/CSS/CSS_box_model/Mastering_margin_collapsing)
+- 包含内部浮动。
+- 排除外部浮动。
+- 阻止[外边距重叠](/zh-CN/docs/Web/CSS/CSS_box_model/Mastering_margin_collapsing)。
 
 > **备注：** flex/grid 容器（{{ cssxref("display") }}：flex/grid/inline-flex/inline-grid）建立新的 flex/grid 格式上下文，除布局之外，它与块格式上下文类似。flex/grid 容器中没有可用的浮动子级，但排除外部浮动和阻止外边距重叠仍然有效。
```
Comment (yin1999) [inline files/zh-cn/web/guide/css/block_formatting_context/index.md:33]:
```suggestion
> **备注：** 弹性/网格容器（{{ cssxref("display") }}：flex/grid/inline-flex/inline-grid）建立新的弹性/网格格式化上下文，除布局之外，它与区块格式化上下文类似。弹性/网格容器中没有可用的浮动子级，但排除外部浮动和阻止外边距重叠仍然有效。
```

**Review by yin1999 — APPROVED** [review]
Thanks, just two suggestions

**Pull request merged**